db/db.js

```diff
@@ -2,22 +2,14 @@
  * This file is used for connecting to MongoDB.
  */
 
-// module.exports = function () {
-//     const mongoose = require('mongoose'),Admin=mongoose.mongo.Admin;
-//     let mongoURL =process.env.MONGO_URL || 'localhost';
-//     let mongoPORT = process.env.MONGO_PORT|| '27017';
-//     let dev_db_url = `mongodb://${mongoURL}:${mongoPORT}/climateTree`;
-//     let mongoDB = process.env.MONGODB_URI || dev_db_url;
-//     mongoose.connect(mongoDB);
-// };
-
 module.exports = function () {
+    const useLocalDB=false;
     if(process.env.DOCKER_ENABLE_CI){
         const mongoose = require('mongoose'),Admin=mongoose.mongo.Admin;
         const mongoPORT = process.env.MONGO_PORT|| '10255';
-        const accountName=process.env.MONGO_ACCOUNT_NAME;
+        const accountName=process.env.MONGO_ACCOUNT_NAME||"stories-mongodb";
```

Comment:
Why do we need this change?
That env variable is already set in production.

Reply:
Because I am not sure it connects to DB correctly. Yesterday, it cannot obtain data from DB.

Reply:
If you are not sure there are ways we can test your hypothesis. If it were true that it does not connect to the db correctly then the deployed instance would never boot up correctly  as it would not connect to db. It would throw errors and crash. I have checked logs and it does not crash you don't need to do this.